releasenotes/notes/context-returns-namedtuple-83bda2334a7fb8d1.yaml

```diff
@@ -0,0 +1,4 @@
+---
+upgrade:
```

Comment:
@arcondello Not sure if this should be under `upgrade`, or something else (e.g., `fix` or `other`).

Reply:
Quacks like a backwards-incompatible new feature (but not because of the `namedtuple`, but because you now return two registers instead of one). So, you can put two entries, one under `features` and one under `upgrade`. But if you want to keep it to one note, the upgrade one is better.

Also, it's nice to include actual upgrade instructions here, like "instead of `with .. as q: q[0]` use `with .. as reg: reg.q[0]`".

Reply:
Ah, good point. Thanks!